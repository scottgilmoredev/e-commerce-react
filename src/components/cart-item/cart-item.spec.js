import React from 'react';
import { shallow } from 'enzyme';

// Component
import CartItem from './cart-item.component';

// Render the component
it('should render the CartItem component', () => {
    const mockItem = {
        imageUrl: 'www.testImage.com',
        name: 'hats',
        price: 10,
        quantity: 2,
    };

    expect(shallow(<CartItem item={ mockItem } />)).toMatchSnapshot();
});
