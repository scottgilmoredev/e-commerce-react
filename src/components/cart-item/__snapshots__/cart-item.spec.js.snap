// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the CartItem component 1`] = `ShallowWrapper {}`;
